packages/ff-bench/ff-bench.0.6.2/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+synopsis: "Benchmark library for finite fields over the package ff-sig"
+description: "Benchmark library for finite fields over the package ff-sig"
+maintainer: "Danny Willems <be.danny.willems@gmail.com>"
+authors: "Danny Willems <be.danny.willems@gmail.com>"
+license: "MIT"
+homepage: "https://gitlab.com/dannywillems/ocaml-ff"
+bug-reports: "https://gitlab.com/dannywillems/ocaml-ff/issues"
+depends: [
+  "ocaml" {>= "4.08"}
+  "dune" {>= "2.7"}
+  "ff-sig" {= version}
+  "core" {= "v0.13.0"}
+  "core_bench" {= "v0.13.0"}
```

Comment:
this does sound a bit too restrictive. Is v0.14 really not supported?
If not could you at least change it to this in case a patched version (e.g. v0.13.1 or even v0.13.0-1) is required in the future?
```suggestion
  "core" {>= "v0.13.0" & < "v0.14"}
  "core_bench" {>= "v0.13.0" & < "v0.14"}
```

Reply:
v0.14 is also fine. I made the changes upstream to `>= "v0.13.0`.